internal/service/advancedcluster/model_advanced_cluster_test.go

```diff
@@ -463,3 +464,185 @@ func TestCheckRegionConfigsPriorityOrder(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertClusterDescription20241023to20240805(t *testing.T) {
```

Comment:
Can we have this test defined in a new file `model_sdk_version_conversion_test`?